packaging/debian-sid/control

```diff
@@ -78,16 +78,25 @@ Architecture: any
 Depends: adduser,
          apparmor (>= 2.10.95-5),
          ca-certificates,
+         default-dbus-session-bus | dbus-session-bus,
          gnupg1 | gnupg,
          openssh-client,
          squashfs-tools,
          systemd,
          udev,
          ${misc:Depends},
-         ${shlibs:Depends},
-         default-dbus-session-bus | dbus-session-bus
-Replaces: ubuntu-snappy (<< 1.9), ubuntu-snappy-cli (<< 1.9), snap-confine (<< 2.23), ubuntu-core-launcher (<< 2.22), snapd-xdg-open (<= 0.0.0)
-Breaks: ubuntu-snappy (<< 1.9), ubuntu-snappy-cli (<< 1.9), snap-confine (<< 2.23), ubuntu-core-launcher (<< 2.22), snapd-xdg-open (<= 0.0.0), ${snapd:Breaks}
+         ${shlibs:Depends}
+Replaces: snap-confine (<< 2.23),
+          snapd-xdg-open (<= 0.0.0),
```

Comment:
out of curiosity, is there a tool to format those files or is it rather a fruit of manual labor? :)